files/ansible/tasks/install_packages.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install dependencies on any Linux system
```

Comment:
Now that this is separated and cleaned, it looks silly on its own. :)
If we're not planning to do anything more substantial than installing packages, I think we can move this inside completely.